io-engine/src/lvm/mod.rs

```diff
@@ -160,6 +164,23 @@ impl PoolOps for VolumeGroup {
     }
 }
 
+#[async_trait::async_trait(?Send)]
+impl BdevStater for VolumeGroup {
+    type Stats = BdevStats;
+
+    async fn stats(&self) -> Result<BdevStats, CoreError> {
+        Err(CoreError::NotSupported {
```

Comment:
this and also in `for LogicalVolume` is where we'll later be calling the lvm stats command directly as we discussed once?

Reply:
Yes correct, we need to collect these stats from lvm.